### PR TITLE
feat: forward txns over p2p

### PR DESCRIPTION
**Pull request by mattsse**
this adds sequencer forwarding if the transaction was received via eth_sendrawtransaction

### CODE REVIEW

File: bin/odyssey/src/main.rs [hunk 1]
```diff
@@ -124,6 +125,12 @@ fn main() {
                 })
                 .await?;
 
+            // spawn raw transaction forwarding
+            let txhandle = node.node.network.transactions_handle().await.unwrap();
+            let raw_txs =
+                node.node.add_ons_handle.eth_api().eth_api().subscribe_to_raw_transactions();
```
Comment (gakonst) [inline bin/odyssey/src/main.rs:131]: `eth_api().eth_api()` we should fix

File: bin/odyssey/src/main.rs [hunk 2]
```diff
@@ -124,6 +125,12 @@ fn main() {
                 })
                 .await?;
 
+            // spawn raw transaction forwarding
+            let txhandle = node.node.network.transactions_handle().await.unwrap();
```
Comment (gakonst) [inline bin/odyssey/src/main.rs:129]: wish we coudl get rid of `node.node`

**Review by gakonst — APPROVED** [review]
LGTM.

File: bin/odyssey/src/main.rs (hunk 1 above)
Comment (mattsse) [reply bin/odyssey/src/main.rs:131]: not unless we want to replace the wrapped `OpEthApi(EthApi)` with actually impls for OpEthApi, but I guess we could rename this fn instead

File: bin/odyssey/src/main.rs (hunk 2 above)
Comment (mattsse) [reply bin/odyssey/src/main.rs:129]: can rename the variable

**Review by onbjerg — APPROVED** [review]

**Pull request merged**